# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/55_metamail-2.7-fixpartial.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mm2.7/src/bin/showpartial.fixpartial	Thu Feb  1 12:20:37 2001
+++ mm2.7/src/bin/showpartial	Thu Feb  1 12:21:29 2001
@@ -13,7 +13,7 @@
 endif
 set file="$1"
 # This next line is because message-id can contain weird chars
-set id=`echo $2 | tr -d  \!\$\&\*\(\)\|\'\"\;\/\<\>\\` 
+set id=`echo $2 | tr -d  \!\$\&\*\(\)\|\'\"\;\/\<\>\\\\` 
 @ partnum = "$3"
 if ($#argv == 3 || "$4" == "") then
     set totalnum=-1
@@ -74,9 +74,9 @@
 end
 if ("$found" == "$totalnum") then
     cd "${TREEROOT}/$id"
-    cat "$list" > "${TREEROOT}/$id/FULL"
+    cat $list > "${TREEROOT}/$id/FULL"
 #    cat ${TREEROOT}/$id/HDRS $list > ${TREEROOT}/$id/FULL
-    rm "$list"
+    rm $list
     echo All parts of this "${totalnum}"-part message have now been read.
     metamail -d "${TREEROOT}/$id/FULL"
     echo WARNING:  To save space, the full file is now being deleted.  
